backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -213,7 +214,7 @@ trait StandardAsyncExecutionActor extends AsyncBackendJobExecutionActor with Sta
     * If the `command` errors for some reason, put a "-1" into the rc file.
     */
   def redirectOutputs(command: String): String = {
-    s"$command > ${jobPaths.stdout} 2> ${jobPaths.stderr} < /dev/null || echo -1 > ${jobPaths.returnCode}"
+    s"""$command > ${jobPaths.stdout} 2> ${jobPaths.stderr} < /dev/null || { rc=$$?; if [ "$$rc" -gt "127" ]; then echo $$rc; else echo -1; fi } > ${jobPaths.returnCode}"""
```

Comment:
ToL what does > 127 mean?

Reply:
What I *meant* for this to do was capture "death by signal", but from the reference below I learned this isn't quite right. Plus I should comment the intent here since it's not exactly obvious.  🙂 

http://www.tldp.org/LDP/abs/html/exitcodes.html